.eggs/README.txt

```diff
@@ -0,0 +1,6 @@
+This directory contains eggs that were downloaded by setuptools to build, test, and run plug-ins.
```

Comment:
I'm not sure how this ended up being included (should have been filtered out via .gitignore), could you please remove the file from the commit?